test/fixtures/constant.js

```diff
@@ -1 +1 @@
-const LOADER = true;
+const LOADER = true; // eslint-disable-line no-unused-vars
```

Comment:
We probably should exclude fixtures completely from eslint, as they could always contain stuff that needs to not respect the rules. What do you think?